.vscode/settings.json

```diff
@@ -12,8 +12,18 @@
 		"**/.next/": true,
 		"**/.git/": true
 	},
+	"editor.defaultFormatter": "esbenp.prettier-vscode",
 	"files.associations": {
 		"*.css": "tailwindcss"
 	},
-	"prettier.documentSelectors": ["**/*.svg"]
+	"editor.quickSuggestions": {
+		"strings": true
+	},
+	"eslint.useFlatConfig": true,
+	"tailwindCSS.experimental.configFile": "tailwind.config.ts",
+	"prettier.documentSelectors": ["**/*.svg"],
+	"sonarlint.connectedMode.project": {
+		"connectionId": "https-sonarqube-sapphirejs-dev",
+		"projectKey": "sapphiredev_resource-webhooks_e2cb9959-cd79-4b35-a323-e26dee4ce7d4"
+	}
```

Comment:
What's this?

Reply:
It's for https://sonarqube.sapphirejs.dev (Sonarqube). I set this up with the free Oracle VM a while ago. TL;DR, code analysis. For example that refactor of chaining replaceAll was a Sonar report. It's a great vscode extension to add.